source/fundamentals/crud/read-operations/sort.txt

```diff
@@ -4,11 +4,199 @@ Sort Results
+
+Ascending
+~~~~~~~~~
+
+An ascending sort orders your results from smallest to largest. To
+specify this sort, pass the field you want to sort by and a ``1`` to the
+``FindOptions.SetSort()`` function.
+
+.. tip::
+
+   With an ascending sort, a ``Boolean`` orders ``false`` **then**
+   ``true``, a ``String`` orders from **a to z** and numeric types order
+   from **negative infinity to positive infinity**.
+
+Example 
+```````
+
+The following example specifies an ascending sort on ``rating`` field:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CRUD/sort.go
+   :language: go
+   :dedent:
+   :start-after: begin ascending sort
+   :end-before: end ascending sort
+   :emphasize-lines: 2-3, 5
+
+After running the preceding example, the output resembles the following:
+
+.. code-block:: none
+   :copyable: false
+
+    //results truncated
+    [_id:ObjectID("...") rating:5 type:Assam]
+    [_id:ObjectID("...") rating:5 type:English Breakfast]
+    [_id:ObjectID("...") rating:7 type:Oolong]
+    [_id:ObjectID("...") rating:8 type:Earl Grey]
+    [_id:ObjectID("...") rating:10 type:Masala]
+
+Descending
+~~~~~~~~~~
+
+A descending sort orders your results from largest to smallest. To
+specify this sort, pass the field you want to sort by and a ``-1`` to the
+``FindOptions.SetSort()`` function.
+
+.. tip::
+
+   With a descending sort, a ``Boolean`` orders ``true`` **then**
+   ``false``, a ``String`` orders from **z to a** and numeric types
+   orders from **positive infinity to negative infinity**.
+
+Example
+```````
+
+The following example specifies a descending sort on ``rating`` field:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/CRUD/sort.go
+   :language: go
+   :dedent:
+   :start-after: begin descending sort
+   :end-before: end descending sort
+   :emphasize-lines: 2-3, 5
+
+After running the preceding example, the output resembles the following:
+
+.. code-block:: none
+   :copyable: false
+
+    [_id:ObjectID("...") rating:10 type:Masala]
+    [_id:ObjectID("...") rating:8 type:Earl Grey]
+    [_id:ObjectID("...") rating:7 type:Oolong]
+    [_id:ObjectID("...") rating:5 type:Assam]
+    [_id:ObjectID("...") rating:5 type:English Breakfast]
+
+Handling Ties
+~~~~~~~~~~~~~
+
+A tie occurs when two or more documents have identical values in the
+field you are using to order your results. MongoDB does not guarantee
+order in the event of ties. 
+
+For example, in the preceding examples, we encounter a tie for the
+``rating`` in the following documents:
+
+.. code-block:: none
+   :copyable: false
+
+   [_id:ObjectID("...") rating:5 type:Assam]
+   [_id:ObjectID("...") rating:5 type:English Breakfast]
+
+To guarantee a specific order for documents that have fields with
+identical values, specify additional fields to sort by in the event of a
+tie.
+
+Example
+```````
+
+The following example specifies an ascending sort on ``rating`` field,
+then a descending sort on the ``type`` field:
```

Comment:
Consider adding to this part: ... then a descending sort on the ``type`` field. 

Such as: ... then a descending sort on the ``type`` field. Sorting by two fields at once allows you to achieve unambiguous sorting results when ties occur for values in one of the fields.